pokedexForm.php

```diff
@@ -0,0 +1,50 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Pokedex Entry</title>
+        <link href="normalize.css" type="text/css" rel="stylesheet">
+        <link href="pokeFormStyle.css" type="text/css" rel="stylesheet">
+    </head>
+    <body class="pokemonFont backgroundImage">
+        <header class="">Add a new Pokemon</header>
+
+        <a href="pokedexDataBaseProject.php" class="backToPokedexButton"><button>Back to Pokedex</button></a>
+
+        <form method="post" action="insertPokemon.php" class="formFormat">
+            URL for Pokemon image:
+            <input type="url" name="addPokemonImage" required>
+            <br>
```

Comment:
dont use br tags, wrap it in a div instead and use margin

Reply:
amended